freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/model/UmsUsersState.java

```diff
@@ -0,0 +1,59 @@
+package com.sequenceiq.freeipa.service.freeipa.user.model;
```

Comment:
Feel free to ignore if this is not the coding convention in this repo, but copywrong statement is missing.